noxfile.py

```
@@ -0,0 +1,325 @@
+            session.log("PostgreSQL is ready.")
+            return
+        time.sleep(1)
+
+    session.error(f"PostgreSQL did not become ready within {POSTGRES_HEALTH_TIMEOUT}s")
+
+
+def _destroy_postgres(session: nox.Session) -> None:
+    """Stop and remove the PostgreSQL container."""
+    if _container_exists(POSTGRES_CONTAINER_NAME):
+        session.log(f"Destroying container '{POSTGRES_CONTAINER_NAME}'...")
+        subprocess.run(["docker", "rm", "-f", POSTGRES_CONTAINER_NAME], check=True)
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)